src/apiCalls.js

```diff
@@ -24,24 +31,34 @@ let apiCalls = {
       .then((data) => data.bookings)
       .catch((error) => console.log(error));
   },
+  getRoomServiceData() {
+    fetch(
+      "https://fe-apps.herokuapp.com/api/v1/overlook/1904/room-services/roomServices"
```

Comment:
I had actually fully forgot about this extensional(?) dataset until now.. how fun!